frontend/src/pages/auth/components/sign-in-form/styles.module.scss

```
@@ -37,3 +37,9 @@
   font-size: 16px;
   font-family: Lato, sans-serif;
 }
+
+.forgot-password-link {
```

Let's follow camelCase in style classes for consistency
```suggestion
.forgotPasswordLink {
```